tools/fpgainfo/errors.c

```diff
@@ -183,6 +183,11 @@ fpga_result errors_filter(fpga_properties *filter, int argc, char *argv[])
 							  FPGA_ACCELERATOR);
 			ON_FPGAINFO_ERR_GOTO(
 				res, out, "setting type to FPGA_ACCELERATOR");
+
+			res = fpgaPropertiesSetInterface(*filter,
```

Comment:
Is this going to restrict the output to *ONLY* ports that are bound to vfio-pci?  That's not what we discussed yesterday.